OneCommunityGlobal/HighestGoodNetworkApp
# Shiwani owner message component lint fix

# Description
Lint fix for OwnerMessage component

## Files changed:
- src/components/OwnerMessage/OwnerMessage.js

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. log in as owner user
4. go to dashboard->click on the edit icon in the header -> add a new message 
5. check if the functionality for OwnerMessage works as before. Comment OwnerMessage folder if not commented before  in eslintignore and check if OwnerMessage component has any errors in the files.
6. Additionally, please check if there are any lint errors when you do npx eslint src/components/OwnerMessage/OwnerMessage.js
7. Feel free to include any comments or screenshots of your test
